React/Profiler/RCTPerfMonitor.m

```diff
@@ -158,17 +158,20 @@ - (RCTDevMenuItem *)devMenuItem
   if (!_devMenuItem) {
     __weak __typeof__(self) weakSelf = self;
```

Comment:
Sounds good. Note that we also need `weakSelf` because we dispatch calls to `show` and `hide` in the block.